internal/gapicgen/execv/gocmd/gocmd.go

```diff
@@ -143,24 +120,4 @@ func Vet(dir string) error {
 	c = execv.Command("gofmt", "-s", "-d", "-w", "-l", ".")
 	c.Dir = dir
 	return c.Run()
-}
-
-// CurrentMod returns the module name of the provided directory.
-func CurrentMod(dir string) (string, error) {
-	log.Println("detecting current module")
-	c := execv.Command("go", "list", "-m")
-	c.Dir = dir
-	var out []byte
-	var err error
-	if out, err = c.Output(); err != nil {
-		return "", err
-	}
-	return strings.TrimSpace(string(out)), nil
-}
-
-// Get upgrades the specified module dependency to the version provided.
-func Get(dir, module, version string) error {
-	c := execv.Command("go", "get", fmt.Sprintf("%s@%s", module, version))
-	c.Dir = dir
-	return c.Run()
-}
+}
```

Comment:
```suggestion
}

```
Need new line at EoF for vet to be happy